scijava/scijava-ops-api/templates/main/java/org/scijava/ops/api/OpBuilder.vm

```diff
@@ -47,13 +47,52 @@ import org.scijava.types.Types;
 
 /**
  * Convenience class for looking up and/or executing ops using a builder
- * pattern.
+ * pattern. The general order of specification:
+ * <ol>
+ *   <li>The op name</li>
+ *   <li>The number of input(s)</li>
+ *   <li>The type or value(s) of input(s)</li>
+ *   <li>One of:
+ *     <ul>
+ *       <li>The type or value of the output</li>
+ *       <li>Which input should be modified in-place</li>
+ *     </ul>
+ *   </li>
+ * </ol>
+ * The first two steps are required, at a minimum. The choices you make will
+ * determine the <i>type</i> of Op that is matched:
+ * <ul>
+ *   <li>No inputs &rarr; <code>Producer</code> or <code>Computer</code></li>
+ *   <li>Inputs with an output <i>value</i> &rarr; <code>Computer</code></li>
+ *   <li>Inputs with an output <i>type</i> &rarr; <code>Computer</code> or <code>Function</code></li>
+ *   <li>Inputs with no output &rarr; <code>Function</code> or <code>Inplace</code></li>
```

Comment:
Maybe change this line to read "Inputs with no output -> Inplace or return-type-unknown Function"?